wolfssl/openssl/pkcs7.h

```diff
@@ -63,6 +98,15 @@ WOLFSSL_API WOLFSSL_STACK* wolfSSL_PKCS7_to_stack(PKCS7* pkcs7);
 WOLFSSL_API WOLFSSL_STACK* wolfSSL_PKCS7_get0_signers(PKCS7* p7,
     WOLFSSL_STACK* certs, int flags);
 WOLFSSL_API int wolfSSL_PEM_write_bio_PKCS7(WOLFSSL_BIO* bio, PKCS7* p7);
+WOLFSSL_API PKCS7* wolfSSL_SMIME_read_PKCS7(WOLFSSL_BIO* in, WOLFSSL_BIO** bcont);
+WOLFSSL_LOCAL int MIME_parse_headers(WOLFSSL_BIO* in, int inLen, MimeHdr** hdrs);
```

Comment:
Please ad `wc_` to front of these new API's. Or remove from header and make the functions `static`.